python/Ganga/Runtime/bootstrap.py

```diff
@@ -319,6 +319,8 @@ def file_opens(f, message):
     @staticmethod
     def new_version():
         versions_filename = os.path.expanduser('~/.cache/Ganga/.used_versions')
+        if not os.path.exists(os.path.dirname(versions_filename)):
```

Comment:
surely you want to be creating the directory not the whole path looking at the code below?
Also, does this work recursively like `mkdir -p` or just create a single empty folder?